src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,181 @@
+package frc.robot.subsystems;
+
+import static com.ctre.phoenix6.signals.NeutralModeValue.Coast;
+import static com.revrobotics.SparkAbsoluteEncoder.Type.kDutyCycle;
+import static edu.wpi.first.units.Units.Volts;
+import static frc.robot.Constants.CANIVORE_BUS_NAME;
+import static frc.robot.Constants.ShooterConstants.DEVICE_ID_ACTUATOR_MOTOR;
+import static frc.robot.Constants.ShooterConstants.DEVICE_ID_SHOOTER_LEFT;
+import static frc.robot.Constants.ShooterConstants.DEVICE_ID_SHOOTER_RIGHT;
+import static frc.robot.Constants.ShooterConstants.SHOOTER_SLOT_CONFIGS;
+import static frc.robot.Constants.ShooterConstants.SHOOTER_VELOCITY_OFFSET;
+import static frc.robot.Constants.ShooterConstants.WRIST_POSITION_OFFSET;
+
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.VelocityTorqueCurrentFOC;
+import com.ctre.phoenix6.controls.VoltageOut;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkAbsoluteEncoder;
+import com.revrobotics.SparkPIDController;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.units.Measure;
+import edu.wpi.first.units.Voltage;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine.Direction;
+import frc.robot.Constants.IndexerConstants;
+import frc.robot.subsystems.sysid.SysIdRoutineSignalLogger;
+
+public class ShooterSubsystem extends SubsystemBase {
+  private final TalonFX shooterLeftMotor = new TalonFX(DEVICE_ID_SHOOTER_LEFT, CANIVORE_BUS_NAME);
+  private final TalonFX shooterRightMotor = new TalonFX(DEVICE_ID_SHOOTER_RIGHT, CANIVORE_BUS_NAME);
+  private final CANSparkMax actuatorMotor = new CANSparkMax(DEVICE_ID_ACTUATOR_MOTOR, MotorType.kBrushless);
+  // creates new devices and sets them to the device Ids set in
+  // Constants.java
+
+  private static final double ENCODER_OFFSET = -0.2285f;
+  private static final float LIMIT_BOTTOM = 0.12f;
+  private static final float LIMIT_TOP = 0.4272f;
+  private static final double LIMIT_TOP_RADIANS = Units.rotationsToRadians(LIMIT_TOP + ENCODER_OFFSET);
+  private static final double LIMIT_BOTTOM_RADIANS = Units.rotationsToRadians(LIMIT_BOTTOM + ENCODER_OFFSET);
+  private boolean isActiveStopped = false;

```

Comment:
I don't think we need an active stop mode, do we? That was for hanging on to cones in Orien, I'm not sure that really applies here.